NASS/transforms/NASS_Injury_Value_Map.py

```diff
@@ -75,7 +101,7 @@
     {
         'ranges': [
             {'start': 1980, 'end': 1980},
-            {'start': 1982, 'end': 1987},
+            {'start': 1982, 'end': 1996},
         ],
         'mapping': {'0': 'Not injured', '1': 'Autopsy records', '2': 'Medical or hospital records',
```

Comment:
Let's make sure the casing is the same for soudat_shared_mapping and the rest of the soudat mapping